### DISCUSSION TITLE
Move website to it's own repo

### DISCUSSION THREAD

**Discussion by privatemaker** [decaporg/decap-cms#7069] [Ideas]
As I have been updating the documentation it comes to my mind- *why does the `website` exist in this already very dense repo containing all the application logic?* In recent years there was this "trend" towards monorepos, and with code projects including multiple related modules a monorepo makes sense vs. git sub-modules.

But why include a whole website which contains documentation and should be regularly updated and maintained? The one argument i've heard (from other projects) is this helps keep the documentation more up-to-date. Nice theory, but in practice (especially that of Decap CMS and [my recent efforts](https://github.com/decaporg/decap-cms/pulls/privatemaker) to update Beta Features) just having the docs pages in the same repo does not help enforce such updating.

The very obvious benefits I see of a separate `website` repo are:

- Smaller repo sizes
- More granular ACL of contributors and maintainers
- Less clutter in the issue tracker and notifications
- Less effort to maintain branch sync with `HEAD` re: to entirely unrelated updates done to the code

Apologies if this is a not-up-for-discussion topic due to history, but it seems like maybe under new leadership this is worth revisting.

**Comment by martinjagodic** [comment]
We've had this discussion internally and it's a valid point. But while it solves the problems that you describe, it also creates new ones.

I believe that at this point the project is still manageable under one repository. It's easier to manage issues, discussions, PRs, and projects under one repo.

I converted this into a discussion because I would like to hear more opinions and experiences from similar projects.

**Reply by privatemaker** [reply]
Ah I see. Care to share what new problems it creates?

**Reply by martinjagodic** [reply]
Mainly it's the confusion of having 2 repos with their own issues system. As much as this is a solution for a large project with many contributors, it's a hassle for projects with small teams. If this project gains more long-term collaborators, it will start to make sense.

It's easier to update docs in the same PR than having to do 2 on 2 repos. You would have to clone both, make PRS, review twice, etc.

It's easier to search locally on one repo. If I want to know about editorial workflow, the docs are up there in my results pane. This is useful for fresh contributors.

It would take some work (not a lot though), but I would rather focus on delivering updates to the core project right now.

There could be more but these come from the top of my mind.

**Comment by martinjagodic** [comment]
@privatemaker you were right: we need a separate repo for the website. We were just releasing a new CMS version and publishing a blog post and it did not go well haha. I will set it up and migrate docs issues there.

**Reply by privatemaker** [reply]
@martinjagodic sorry to hear about the clusterduck 💥 🦆 but glad to hear the website will live independently. In the case you don't know about the trick to maintain history of one directory, I will [link it here](https://medium.com/@ayushya/move-directory-from-one-repository-to-another-preserving-git-history-d210fa049d4b)

**Reply by martinjagodic** [reply]
I didn't know about this, thanks!

**Comment by privatemaker** [comment]
This has been resolved as the site now lives in the repo: [decap-website](https://github.com/decaporg/decap-website) 🥳 🎈
